src/components/author/author.twig

```diff
@@ -0,0 +1,21 @@
+<div class="c-author o-media{% if class %} {{class}}{% endif %}">
+  {% embed '@cloudfour/objects/bunch/bunch.twig' with { class: "o-media__object" } %}
+    {% block content %}
+      {% for author in authors %}
+        {% include '@cloudfour/components/avatar/avatar.twig' with { src: author.avatar() } only %}
+      {% endfor %}
+    {% endblock %}
+  {% endembed %}
+  <div class="o-media__content">
+    <div>
+      {% for author in authors %}
+        {% if loop.last and loop.length > 1 %}and {% endif %}
+        <a href="{{ author.link() }}">{{ author.name() }}</a>
+        {%- if not loop.last and loop.length > 2 %},{% endif %}
+      {% endfor %}
+    </div>
```

Comment:
I mentioned this in my [last comment](https://github.com/cloudfour/cloudfour.com-patterns/pull/1244#issuecomment-852561426), but figured I'd add something here too. 🙂 

Could we add visually hidden text to provide more context? 

Ideas include:
- Authored by `<name>`, `<name>`
- Authors: `<name>`, `<name>`

Ideally, the rendered output is something like:
```html
<div>
  <span class="u-hidden-visually">Authored by</span>
  <a href="https://cloudfour.com/is/megnotarte">Megan Notarte</a>,
  <a href="https://cloudfour.com/is/gerardo">Gerardo Rodriguez</a>,
  <a href="https://cloudfour.com/is/tyler">Tyler Sticka</a>
</div>
```

Reply:
Yep! Already done! [`c616e9e`](https://github.com/cloudfour/cloudfour.com-patterns/pull/1244/commits/c616e9e57b9b2431d6cd42f9cda251b0e2140b18) 🎉